code/modules/shuttle/computers/dropship_computer.dm

```diff
@@ -264,6 +292,18 @@
 			to_chat(xeno, "The shuttle is launching.")
 			return
 
+/obj/structure/machinery/computer/shuttle/dropship/flight/proc/alien_call_dropship(mob/living/carbon/xenomorph/xeno, obj/docking_port/mobile/shuttle, obj/docking_port/stationary/marine_dropship/airlock/inner/inner_airlock)
+	var/result = SSshuttle.moveShuttle(shuttleId, linked_lz, TRUE)
+	if(result != DOCKING_SUCCESS)
+		to_chat(xeno, SPAN_WARNING("The metal bird can not land here. It might be currently occupied!"))
+		return
+	if(inner_airlock)
+		inner_airlock.disable_manual_input = FALSE
+	to_chat(xeno, SPAN_NOTICE("You command the metal bird to come down. Clever girl."))
```

Comment:
As far as im concerned we've moved away or have been trying to move away from stuff like "metal bird" or "metal turtle" probably turning this into just the dropship is better